swift-lang/src/main/resources/ecocode_swift_profile.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "ecoCode",
   "ruleKeys": [
-    "EIDL001",
-    "EIDL002",
-    "EPOW001",
-    "EPOW002",
-    "ESOB001",
-    "ESOB002",
-    "ESOB003",
-    "ESOB005",
-    "ESOB006",
+    "EC14",
```

Comment:
why ECXX and not ECXXX ? I would think that rules for mobile was EC5XX or EC6XX, wouldn't it ?

Reply:
Ok I see...
rules are duplicated inside the common repository :(
Rule EC14 is also EC505...

Reply:
And it seems that EC602 is not published yet... (that is why the build is breaking)

Reply:
I created a PR to clean  and add missing rule here: https://github.com/green-code-initiative/ecoCode/pull/280
Once released, everything should be ok !